### DISCUSSION TITLE
I want to create a custom tag-like property

### DISCUSSION THREAD

**Discussion by patrykpalej** [cotes2020/jekyll-theme-chirpy#1401] [Q&A]
### Checklist

- [x] I have read and accepted the [contributing guidelines](https://github.com/cotes2020/jekyll-theme-chirpy/blob/master/docs/CONTRIBUTING.md).

### Is your feature request related to a problem? Please describe

I want to create a custom property which will act just like tags, but regardless of them. Something like "categories", but in fact they will be just a copy of "tags". We can call it `mytags`.

In general it can be anything that somehow categorizes posts.

It's probably possible, but I cannot obtain this.

### Describe the solution you'd like

I created:
- `mytags.md` in `_tabs` folder and filled it analogously to `tags.md`
- `mytags.html` in `_layouts`. It is actually a copy of `tags.md` but `site.tags` needed to be renamed. Actually how do you define `site.<something>`?
- updated _config.yml similarly to tags and categories in that config

I cannot figure it out how to gather the property values from all the posts and create subfolders in `_site/mytags`. There is only an `index.html` in that folder.

### Additional context

I also considered having two separate blog-like tabs. The posts could all be stored in the same directory but there would be a property which determines which of these two the post belongs to. I tried doing this but the posts in the second tab don't display properly, probably because of pagination issues.

Would it involve modifying the underlying ruby gem or it can be done just by editing the template?

**Comment by cotes2020** [comment]
I'm not going to describe exactly how this should be done, but you can learn about Jekyll's **Collections** to accomplish this. 

See: https://jekyllrb.com/docs/collections/

**Reply by patrykpalej** [reply]
Thank you, I'll read about it. Can you just tell me if it can be obtained just by modifying theme's files or I need to deal with the gem code as well?
